src/vimhelp/html.clj

```diff
@@ -2,20 +2,27 @@
   (:require
    [clojure.java.io :as io]
    [clojure.string :as str]
-   [hiccup.core :as hiccup]
-   [hiccup.page :as page])
+   [hiccup2.core :as hiccup])
   (:import
    java.net.URLEncoder))
 
 (defn- url-encode
   [s]
   (URLEncoder/encode s "UTF-8"))
 
-(defn- replace-spaces
+(defn replace-spaces
```

Comment:
I don't like making function public for testing.
You can use `#'sut/replace-spaces` instead.

Reply:
Oh I didn't know you could do that :o